### PR TITLE
New rule: use_expression_function_body_when_possible

### PR DESCRIPTION
**Pull request by JPaulsen**
Fixes https://github.com/dart-lang/linter/issues/463

### CODE REVIEW

File: lib/src/rules/use_equal_and_higher_in_single_return_statement.dart [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+containsValue(String value) => getValues().contains(value);
+```
+
+''';
+
+class UseEqualAndHigherInSingleReturnStatement extends LintRule {
+  _Visitor _visitor;
+  UseEqualAndHigherInSingleReturnStatement()
+      : super(
+            name: 'use_equal_and_higher_in_single_return_statement',
```
Comment (bwilkerson) [inline lib/src/rules/use_equal_and_higher_in_single_return_statement.dart]: Perhaps "use_expression_function_body_when_possible" or even "use_expression_function_body".

Comment (JPaulsen) [inline lib/src/rules/use_equal_and_higher_in_single_return_statement.dart]: Fixed

File: test/rules/use_expression_function_body_when_possible.dart [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    }
+    return 0;
+  }
+
+  get badWidth { // LINT
+    return right - left;
+  }
+
+  get goodWidth => right - left; // OK
+}
```
Comment (bwilkerson) [inline test/rules/use_expression_function_body_when_possible.dart]: Might be good to add a test involving a setter.

Comment (JPaulsen) [inline test/rules/use_expression_function_body_when_possible.dart]: Fixed

**Pull request merged**